docs/howto/odk-add-orcidio-module.md

```diff
@@ -0,0 +1,87 @@
+## Adding an ORCIDIO import to your ontology with ODK
+
+[ORCIDIO](https://github.com/cthoyt/orcidio) is an ontology that integrates ORCIDs with basic metadata such as the indentified individuals names.
+
+When attributing a term to an individual using their ORCID, one of the problems is that it is not immediately obvious which ORCID belongs to which person.
+ORCIDIO solves this issue by labelling each ORCID using the identified individuals names. Tools like Protege will then display that name to 
+the user rather than the ORCID itself, which makes it immediately more readable. Here we discuss how to add ORCIDO to your ODK setup.
+
+#### 1. Include ORCIDIO as an import into the ODK config file
+
+In your ODK config (e.g. `src/ontology/myont-odk.yaml`), add the following to the `import_group`:
+
+```
+import_group:
+  annotation_properties:
+    - rdfs:label
+    - dc:description
+    - dc:source
+    - IAO:0000115
+  products:
+    - id: orcidio
+      mirror_from: https://w3id.org/orcidio/orcidio.owl
+      module_type: filter
+      base_iris:
+        - https://orcid.org/
+```
+
+The list of annotation properties, in particular `dc:source`, is important for the `filter` module to work (ORCIDIO relies heavily on axiom annotations for provenance).
+
+#### 2. Update your catalog
+
+As usual, add a statement into your catalog (`src/ontology/catalog-v001.xml`):
+
+```
+ <uri name="http://purl.obolibrary.org/obo/ro/imports/orcidio_import.owl" uri="imports/orcidio_import.owl"/>
+ ```
+ 
+ #### 3. Update the edit file
+ 
+ As usual, add an imports declaration to your edit file (`src/ontology/myont-edit.owl`):
+ 
+ ```
+ Import(<http://purl.obolibrary.org/obo/ro/imports/orcidio_import.owl>)
+ ```
+ 
+ Note: This is not necessary when using the `base merging` strategy (you will know what this means when you do use it).
+ 
+ #### 4. Configure your _seed_:
+ 
+ Add a new SPARQL query: `src/sparql/orcids.sparql`. This is used to query for all ORCIDs used in your ontology.
+ 
+ ```
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+SELECT DISTINCT ?orcid
+WHERE {
+  VALUES ?property { <http://purl.org/dc/elements/1.1/creator>  <http://purl.org/dc/elements/1.1/contributor> <http://purl.org/dc/terms/creator> <http://purl.org/dc/terms/contributor> }
+  ?term ?property ?orcid . 
+  FILTER(isIRI(?term))
+}
+ ```
+ 
```

Comment:
```suggestion
 
 For your specific use-case, it may be necessary to tweak this SPARQL query, for example if your ORCIDs are used on axiom annotation level rather than entity annotation level.
 
```